test/addons/stringbytes-external-exceed-max/test-stringbytes-external-exceed-max.js

```diff
@@ -5,6 +5,9 @@ const skipMessage = 'intensive toString tests due to memory confinements';
 if (!common.enoughTestMem)
   common.skip(skipMessage);
 
+if (process.platform === 'aix')
+  common.skip('flaky on AIX')
```

Comment:
Maybe reference  `https://github.com/nodejs/build/issues/1820#issuecomment-505998851` as has been done for the skipped message test?